src/hacksteader.rs

```diff
@@ -693,13 +693,21 @@ impl Plant {
 }
 
 #[derive(Clone, Debug)]
+<<<<<<< HEAD
```

Comment:
Resolve the conflicts, don't commit them

Reply:
lmao I'm an idiot, will fix tomorrow morning.

> Resolve the conflicts, don't commit them

Great advice, I'll be sure to take it into consideration in the future